server/game/cards/01_SOR/events/SneakAttack.ts

```diff
@@ -0,0 +1,39 @@
+import { EventCard } from '../../../core/card/EventCard';
+import AbilityHelper from '../../../AbilityHelper';
+import { CardType, PhaseName, ZoneName } from '../../../core/Constants';
+import { CostAdjustType } from '../../../core/cost/CostAdjuster';
+
+export default class SneakAttack extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3426168686',
+            internalName: 'sneak-attack'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Play a unit from your hand. It costs 3 less and enters play ready. At the start of the regroup phase, defeat it.',
+            immediateEffect: AbilityHelper.immediateEffects.selectCard({
+                activePromptTitle: 'Play a unit, it costs 3 less and enters play ready. At the start of the regroup phase, defeat it.',
+                cardTypeFilter: CardType.BasicUnit,
+                zoneFilter: ZoneName.Hand,
```

Comment:
Let's add `controller: RelativePlayer.Self` for now, we're in the middle of reworking some of the play-from-hand flow and it may cause issues if we don't have it